modules/webbrowser/CMakeLists.txt

```diff
@@ -158,7 +158,11 @@ set(WEBBROWSER_RESOURCES_SRCS
 
 # Place Helper in separate executable
 # The naming style "<ApplicionName> Helper" is required by Chromium.
-set(CEF_HELPER_TARGET "OpenSpace Helper" CACHE INTERNAL "CEF_HELPER_TARGET")
+if (UNIX)
+  set(CEF_HELPER_TARGET "OpenSpace_Helper" CACHE INTERNAL "CEF_HELPER_TARGET")
```

Comment:
Would be good to have a comment here that explains that the underscore is not an error, but intentional

Reply:
Added a comment, and also restricted this to Linux _and_ GNU Make builds